src/lib/dux/sdk/actionTypes.ts

```diff
@@ -1,12 +1,18 @@
-import { ObjectValues } from "../../../utils/typeHelpers/objectValues";
+import SendbirdChat from "@sendbird/chat";
+import { CreateAction } from "../../../utils/typeHelpers/reducers/CreateAction";
 
-const actionTypes = {
+export const SDK_ACTIONS = {
```

Comment:
I am not big fan of enums, they make life hard for customer 

With this pattern, both `<Sendbird logLevel={LOGGER.ERROR} />` and  `<Sendbird logLevel="error" />` works definitively(with out ambiguity)